api.planx.uk/modules/flows/validate/validate.test.ts

```diff
@@ -54,7 +54,7 @@ beforeEach(() => {
     matchOnVariables: false,
     data: {
       flow: {
-        publishFlows: [
+        publishedFlows: [
```

Comment:
Typo in mock queries was causing other tests to fail with the updated logic. This now correctly
 matched the `GetMostRecentPublishedFlowVersion` query.

https://github.com/theopensystemslab/planx-new/blob/main/api.planx.uk/helpers.ts#L225-L234